lib/modules/manager/maven-wrapper/extract.ts

```diff
@@ -15,10 +15,17 @@ const WRAPPER_URL_REGEX = regEx(
   '^(?:wrapperUrl\\s*=\\s*)(?<url>\\S*-(?<version>\\d+\\.\\d+(?:\\.\\d+)?(?:-\\w+)*)(?:.jar))',
 );
 
+const WRAPPER_VERSION_REGEX = regEx(
+  '^(?:wrapperVersion\\s*=\\s*)(?<url>\\S*-(?<version>\\d+\\.\\d+(?:\\.\\d+)?(?:-\\w+)*)(?:.jar))',
```

Comment:
The value of the `wrapperVersion` property should be just a version number, so for example: `wrapperVersion=3.3.0`. This shouldn't need any regular expression or other parsing.

Reply:
it seems we only need the version group? 🤔

Reply:
yeah so I pushed an update but that will leave the url empty,
which I think is no problem because `wrapper:wrapper` will do the work.

But I'm not sure how we are going to make this work (and if we should). I'm in no way a maven-wrapper expert so I will discuss with Nils if this a problem.

```
function getCustomMavenWrapperRepoUrl(
  deps: PackageDependency[],
): string | null {
  const replaceString = deps.find(
    (dep) => dep.depName === 'maven-wrapper',
  )?.replaceString;

  if (!replaceString) {
    return null;
  }

  const match = regEx(/^(.*?)\/org\/apache\/maven\/wrapper\//).exec(
    replaceString,
  );

  if (!match) {
    return null;
  }

  return match[1] === DEFAULT_MAVEN_REPO_URL ? null : match[1];
}
```

Reply:
since it will give that as option to the maven command:

```
function getExtraEnvOptions(deps: PackageDependency[]): ExtraEnv {
  const customMavenWrapperUrl = getCustomMavenWrapperRepoUrl(deps);
  if (customMavenWrapperUrl) {
    return { MVNW_REPOURL: customMavenWrapperUrl };
  }
  return {};
}
```